### PR TITLE
MG-2060 - Remove int32 type in policy interface

### LINKED ISSUE #2060: Remove int32 type in Policy agent interface

### PR DESCRIPTION
**Pull request by nyagamunene**
# What type of PR is this?
This is a refactor because it changes the integer type of  limit and count parameters in policyAgent and Authz interface . 

## What does this do?
It changes the the limit and count parameter in policyAgent and Authz interface from int32 to uint64.

## Which issue(s) does this PR fix/relate to?

- Resolves #2060 

## Have you included tests for your changes?
Yes. I have include test for my changes.

## Did you document any new/modified feature?
No.

### CODE REVIEW

File: auth/api/grpc/server.go [hunk 1]
```diff
@@ -423,7 +423,7 @@ func decodeListObjectsRequest(_ context.Context, grpcReq interface{}) (interface
 		ObjectType:    req.GetObjectType(),
 		Object:        req.GetObject(),
 		NextPageToken: req.GetNextPageToken(),
-		Limit:         req.GetLimit(),
+		Limit:         uint64(req.GetLimit()),
```
Comment (rodneyosodo) [inline auth/api/grpc/server.go]: Change the `limit` in the proto file to `uint64` too.

**Review by rodneyosodo — CHANGES_REQUESTED** [review]

File: auth/api/grpc/server.go (hunk 1 above)
Comment (nyagamunene) [reply auth/api/grpc/server.go]: It raises an issue during the test hence it doesn't pass it

Comment (dborovcanin) [reply auth/api/grpc/server.go]: Please fix the tests as well, in that case.

**Review by dborovcanin — CHANGES_REQUESTED** [review]
Please fix CI.

File: auth.proto [hunk 2]
```diff
@@ -56,7 +56,7 @@ message IdentityRes {
 message IssueReq {
   string user_id = 1;
   optional string domain_id = 2;
-  uint32 type = 3; 
+  uint64 type = 3;
```
Comment (rodneyosodo) [inline auth.proto]: change this to 
```suggestion
  uint32 type = 3; 
```

**Review by rodneyosodo — CHANGES_REQUESTED** [review]

**Review by rodneyosodo — APPROVED** [review]
LGTM

**Review by arvindh123 — APPROVED** [review]
I have manually ran basic test for this PR, LGTM

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**